src/test/end-to-end/pages/amendClaimantRepresentative/amendClaimantRepresentative.js

```diff
@@ -5,13 +5,42 @@ const testConfig = require("../../../config");
 module.exports = async function () {
 
     const I = this;
-    await I.click(commonConfig.claimantRepresentativeYes);
-    await I.fillField('#representativeClaimantType_name_of_representative', commonConfig.claimantRepresentativeName);
-    await I.fillField('#representativeClaimantType_representative_address_representative_address_postcodeInput', commonConfig.claimantRepresentativePostCode);
-    await I.click(commonConfig.findAddressButton);
-    await I.wait(3);
+    I.waitForText('Claimant Representative', testConfig.TestTimeToWaitForText);
+    I.see('Case Number:')
+    I.see('Is the Claimant Represented? (Optional)');
+    I.see('Yes');
+    I.see('No');
+    I.click(commonConfig.claimantRepresentativeYes);
+
+    I.see('Claimant Representative Details');
+    I.see('Name of representative');
+    I.see('Name of Organisation (Optional)');
+    I.see('Occupation (Optional)');
+    I.see('Reference (Optional)');
+    I.see('Phone number (Optional)');
+    I.see('Alternative number (Optional)');
+    I.see('Email address (Optional)');
+    I.see('Contact preference (Optional)');
+    I.see('Address');
+    I.see('Enter a UK postcode');
+
+    I.fillField('#representativeClaimantType_name_of_representative', commonConfig.claimantRepresentativeName);
+    I.fillField('Name of Organisation (Optional)', 'Organisation');
+    I.selectOption('Occupation (Optional)', 'Solicitor');
+    I.fillField('Reference (Optional)', 'ReferenceOptional');
+    I.fillField('Phone number (Optional)', '07912345678');
+    I.fillField('Alternative number (Optional)', '07998765432');
+    I.fillField('Email address (Optional)', 'test@email.com');
+    I.selectOption('Contact preference (Optional)', 'Email');
+    
+    I.fillField('#representativeClaimantType_representative_address_representative_address_postcodeInput', commonConfig.claimantRepresentativePostCode);
+    I.click(commonConfig.findAddressButton);
     I.waitForText(commonConfig.claimantRepresentativeAddress, testConfig.TestTimeToWaitForText);
-    await I.selectOption('#representativeClaimantType_representative_address_representative_address_addressList', commonConfig.claimantRepresentativeAddress);
-    await I.click(commonConfig.continue);
-    await I.click(commonConfig.submit)
+    I.selectOption('#representativeClaimantType_representative_address_representative_address_addressList', commonConfig.claimantRepresentativeAddress);
+    I.click(commonConfig.continue);
+
+    I.waitForText('Claimant Representative', testConfig.TestTimeToWaitForText);
+    I.see('Case Number:');
+    I.click(commonConfig.submit);
+    I.see('has been updated with event: Claimant Representative');
```

Comment:
Can you please Click on the claimant Representative Tab and Check all the relevant Input values are present..?

Reply:
Added tab and verified input values